solr/solrj/src/java/org/apache/solr/common/cloud/PerReplicaStates.java

```diff
@@ -278,6 +278,10 @@ private State insert(State duplicate) {
       }
     }
 
+    public State getDuplicate() {
```

Comment:
What is the purpose of it?

Reply:
just in case if there are duplicate states left over. unlikely, but being not impossible